swift/extractor/remapping/SwiftOpenInterception.macOS.cpp

```diff
@@ -27,7 +27,7 @@ static std::string originalHashFile(const fs::path& filename) {
 }
 
 static int codeql_open(const char* path, int oflag, ...) {
-  va_list ap = {0};
+  va_list ap;
```

Comment:
why was this changed?

Reply:
It was showing up as a warning, so I just piggybacked the fix on this PR.